index.css

```diff
@@ -76,3 +74,162 @@ body {
 .social-icons li {
   list-style-type: none;
 }
+
+/* ----------Works page styling begins here----- */
+
```

Comment:
- [ ] Kindly consider styling the Section of works with `display: grid` as it is a project requirement.
- The main container of the projects has a `display:grid`